fitbenchmarking/controllers/base_controller.py

```diff
@@ -293,12 +293,10 @@ def check_attributes(self):
                             'list or numpy ndarray of floats, got '
                             '{}'.format(attr_name, attr))
 
-    @abstractmethod
     def jacobian_information(self):
```

Comment:
I think we should just make this a list in the class (like `algorithm_check`). The bool can probably also be removed if we make the list show the enabled ones, as in the hessian func, as that is the same as checking if the list is empty (`if controller.hessian_enabled_solvers`)